api/Window.json

```diff
@@ -10069,55 +9924,6 @@
           }
         }
       },
-      "vrdisplayblur_event": {
```

Comment:
Seems like this was in Edge: https://docs.microsoft.com/en-us/previous-versions/mt801971(v%3Dvs.85)

I'll send a PR to copy the data from the onvrdisplayblur" entry.